### PR TITLE
Header

### PR DESCRIPTION
**Pull request by MichiBerger**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,9 +13,15 @@
     <h1>Michi Berger</h1>
     <nav>
       <a href="#contact">Contact</a>
+      <a href="#biography">Biography</a>
```
Comment (badabam) [inline index.html:16]: Nice!

File: index.html [hunk 2]
```diff
@@ -13,9 +13,15 @@
...
     </nav>
   </header>
-  <section id="contact"></section>
+  <section id="contact">
+    <h2>Contact</h2>
+  </section>
+  <section id="biography">
+    <h2>Biography</h2>
```
Comment (badabam) [inline index.html:23]: I would have done the same! Haha!

**Review by badabam — APPROVED** [review]
Good job!

File: index.html (hunk 1 above)
Comment (MichiBerger) [reply index.html:16]: yeah thx ;)

**Pull request merged**